From 03c46d4ea3da0977ce4625ed281c1758bb633664 Mon Sep 17 00:00:00 2001
From: Jonathan Nieder <jrnieder@gmail.com>
Date: Tue, 4 Nov 2014 12:38:58 -0800
Subject: record version

Generated using GIT-VERSION-GEN and git-gui/GIT-VERSION-GEN.

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 git-gui/version | 1 +
 version         | 1 +
 2 files changed, 2 insertions(+)
 create mode 100644 git-gui/version
 create mode 100644 version

diff --git a/git-gui/version b/git-gui/version
new file mode 100644
index 0000000..0b31b05
--- /dev/null
+++ b/git-gui/version
@@ -0,0 +1 @@
+0.19.0.2.g3decb8e
diff --git a/version b/version
new file mode 100644
index 0000000..4cd4bad
--- /dev/null
+++ b/version
@@ -0,0 +1 @@
+2.2.0.rc0.207.ga3a616c
-- 
2.1.0.rc2.206.gedb03e5

